lang/pl_PL.json

```diff
@@ -0,0 +1,313 @@
+    "There are no copyright-protected ROMs available for download on RetroAchievements.org.": "Nie ma żadnych chronionych prawem autorskim ROM-ów dostępnych do pobrania na stronie RetroAchievements.org.",
+    ":earned of :total achievements unlocked": ":earned z :total osiągnięć odblokowanych",
+    ":earned of :total softcore achievements unlocked": ":earned z :total osiągnięć softcore odblokowanych",
+    ":earned of :total points earned": ":earned z :total punktów zdobytych",
+    ":earned of :total softcore points earned": ":earned z :total punktów softcore zdobytych",
+    "Finished": "Ukończone",
+    "Mastered": "Wymasterowane",
+    "Completed": "Zakończone",
+    "Beaten": "Ukończone",
+    "Beaten (softcore)": "Ukończone (softcore)",
+    ":awardLabel on :awardDate": ":awardLabel on :awardDate",
+    "RetroPoints: A measurement of rarity and estimated difficulty.": "RetroPoints: Miara rzadkości i szacowana trudność.",
+    "Derived from points, number of achievers, and number of players.": "Pochodzi z punktów, liczby odblokowanych i liczby graczy.",
+    "All Games": "Wszystkie gry",
+    ":achievementTitle - Report Issue": ":achievementTitle - Zgłoś błąd",
+    "Report Issue - :achievementTitle": "Zgłoś błąd - :achievementTitle",
+    "What sort of issue would you like to report?": "Jaki rodzaj błędu chcesz zgłosić?",
+    "The achievement type (progression/win/missable) is not correct.": "Typ osiągnięcia (postęp/wygrana/pomijalny) nie jest poprawny.",
+    "There is a spelling or grammatical error in the title or description.": "Występuje literówka lub błąd gramatyczny w tytule lub opisie.",
+    "I have an issue with this achievement that is not described above.": "Mam problem z tym osiągnięciem inny niż opisane powyżej.",
+    "I met the requirements, but the achievement did not trigger.": "Wymagania zostały spełnione, ale osiągnięcie nie zostało wywołane.",
+    "I unlocked this achievement without meeting the requirements, and then I reset it.": "Osiągnięcie zostało odblokowane, pomimo niespełnienia wymagań, następnie je zresetowałem/am.",
+    "The achievement triggered, but the unlock didn't appear on my profile.": "Osiągnięcie zostało wywołane, ale odblokowane nie pokazuje się na moim profilu.",
+    "I unlocked this achievement without meeting the requirements.": "Osiągnięcie zostało odblokowane, pomimo niespełnienia wymagań.",
+    "Yes": "Tak",
+    "No": "Nie",
+    "Either": "Oba",
+    "System": "System",
+    "Has achievements": "Ma osiągnięcia",
+    "Reset": "Zresetuj",
+    ":count of :total game|:count of :total games": ":count z :total gier|:count z :total gier",
+    ":count game|:count games": ":count gra|:count gier",
+    ":count selected": ":count wybranych",
+    "No options found.": "No options found.",
```

Comment:
Would "Nie znaleziono opcji." fit here? This is for when the user searches for a filter option, and no filter option is found for their search.

Reply:
This was one of the things I wanted to ask about, to learn more context. Now I see it's in "Watn To Play" when using "System" dropdown search.
I think this translation would fit in there.